plugins/Marketplace/vue/src/StartFreeTrialDialog/StartFreeTrialDialog.vue

```diff
@@ -0,0 +1,107 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="modal" id="trialRequiresLicense">
+    <p class="btn-close modal-close"><i class="icon-close"></i></p>
+    <div class="modal-content">
+      <div class="modal-text">
+        <h2>Start your free trial today</h2>
+        <p>To unlock the full potential of our premium plugins,
+          <a
+            :href="linkTo({'module':'Marketplace', 'action':'manageLicenseKey'})"
+          >add your license key</a>.</p>
+        <p>
+          <strong>Don't have a license key yet?</strong>
+          Visit our <a href="https://shop.matomo.org/my-account/"
+                       rel="noopener noreferrer" target="_blank">online marketplace</a>,
```

Comment:
Should this use the directive that we have for adding the tracking params to in-app links?